fdbserver/storageserver.actor.cpp

```diff
@@ -3384,13 +3393,17 @@ ACTOR Future<Void> update(StorageServer* data, bool* pReceivedUpdate) {
 			eager = UpdateEagerReadInfo();
 		}
 
+		// TODO: Why is is named SSSlowTakeLock2?
```

Comment:
The name could probably be more descriptive, but there is a `SSSlowTakeLock1`.

Reply:
I mean, I don't think this is instrumenting the time of taking any lock. It's instrumenting the time spent to do eager reads, right?

Reply:
Missed that. Yeah, that's correct. Thanks!